### DISCUSSION TITLE
git clone https://github.com/CoolKit-Technologies/ha-addon.git Not Working

### DISCUSSION THREAD

**Discussion by filmnight** [community/community#133225] [Programming Help] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

git clone https://github.com/CoolKit-Technologies/ha-addon.git Not Woking

Below is what i get when pasted into Manage Addon Repositories

Cmd('git') failed due to: exit code(128) cmdline: git clone -v --recursive --depth=1 --shallow-submodules -- https://github.com/Coolkit_technologies/ha-addon /data/addons/git/a4715dfe stderr: 'Cloning into '/data/addons/git/a4715dfe'... fatal: could not read Username for 'https://github.com': No such device or address '

**Comment by praneeth-balamurugan** [comment]
clear the cache and try again

**Reply by filmnight** [reply]
Hi which cache please be patient very new to home assistant and git hub

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @filmnight ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.

**Comment by CleverLemming1337** [comment]
Have you checked your internet connection?

**Reply by filmnight** [reply, marked as answer]
Hi problem was reolved a while ago thank you anyway

**Comment by cyndie-lauper** [comment]
i think u should try using an SSH key instead of HTTPS as usual.
